packages/editor-ui/src/components/NodeWebhooks.vue

```diff
@@ -94,14 +97,48 @@ export default defineComponent({
 				(webhookData) => webhookData.restartWebhook !== true,
 			);
 		},
+		showHttpLabel(): boolean {
+			return this.nodeType.name !== FORM_TRIGGER_NODE_TYPE;
```

Comment:
could also be another flag similar to `hidden`

Reply:
I would say it's too specific for UI

Reply:
hidden is pretty UI specific too no? just hiding the input